dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -414,4 +421,8 @@ bool FramesDecoder::ReadNextFrame(uint8_t *data, bool copy_to_output) {
   }
   return ReadFlushFrame(data, copy_to_output);
 }
+
```

Comment:
How about adding:
```
std::vector<IndexEntry> &FramesDecoder::Index() {
  return index_.value();
}

Reply:
I added the other wrapper to check `has_value` and thorw, if functionality is unavailable (like `Seek`). 
I access `index_` directly only in index building, where we assume the index is there and these are all uses of the index. If the need for another wrapper arises in the future, I'll add it, ok?

Reply:
Sure. It is just an idea.